src/shared/containers/DashboardResultsContainer.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import { useNexus } from '@bbp/react-nexus';
+import { Spin, Alert } from 'antd';
+import ResulTable from '../components/ResultsTable/ResultsTable';
+
+export type Binding = {
+  [key: string]: {
+    dataType?: string;
+    type: string;
+    value: string;
+  };
+};
+
+type Item = {
+  id: string;
+  self: string;
+  key: string;
+  [key: string]: any;
+};
+
+const DashboardResultsContainer: React.FunctionComponent<{
+  dataQuery: string;
+  orgLabel: string;
+  projectLabel: string;
+  viewId: string;
+  handleClick: (self: string) => void;
+}> = props => {
+  const { loading, data, error } = useNexus<any>(
+    nexus =>
+      nexus.View.sparqlQuery(
+        props.orgLabel,
+        props.projectLabel,
+        encodeURIComponent(props.viewId),
+        props.dataQuery
+      ),
+    [props.dataQuery, props.viewId] // only trigger new call if we have a new query and a new view
```

Comment:
There are a lot of API calls being made when changing the dashboard tab. You can simply remove those 2 args from the array it the API call will only be triggered once
```suggestion
    []
```